addons/event-audit/common/src/main/java/org/commonjava/indy/event/audit/conf/EventAuditConfig.java

```diff
@@ -0,0 +1,54 @@
+package org.commonjava.indy.event.audit.conf;
+
+import org.commonjava.indy.conf.IndyConfigInfo;
+import org.commonjava.web.config.annotation.ConfigName;
+import org.commonjava.web.config.annotation.SectionName;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.File;
+import java.io.InputStream;
+
+@SectionName("event-audit")
+@ApplicationScoped
+public class EventAuditConfig implements IndyConfigInfo
+{
+
+    public static final boolean DEFAULT_ENABLED = true;
```

Comment:
Default here should be false, I think. Think of this as a feature flag, and we won't be ready to start publishing these events until the whole architecture is completely settled, with the auditquery service more or less ready to deploy. We can start gathering information a little before then, but we will still want to ensure the data types are settled.

Reply:
Sure.